LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/signal-abort-intercept-expected.txt

```diff
@@ -1,3 +1,5 @@
-FAIL: Timed out waiting for notifyDone to be called
 
+Harness Error (TIMEOUT), message = null
```

Comment:
Should we also skip the tests that have newly become timeouts?

Reply:
If you mean this one,  both the old and new expectations look like timeouts to me, though I am not quite sure what parts of the testing infrastructure cause them exactly (the new one may be triggered by the wpt  test framework if I have to guess).